addons/mail/static/src/components/activity/activity.xml

```diff
@@ -2,52 +2,55 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.Activity" owl="1">
-        <div class="o_Activity" t-attf-class="{{ className }}" t-on-click="activityView.onClickActivity" t-ref="root">
+        <div class="o_Activity d-flex p-2" t-attf-class="{{ className }}" t-on-click="activityView.onClickActivity" t-ref="root">
             <t t-if="activityView">
-                <div class="o_Activity_sidebar">
-                    <div class="o_Activity_user">
+                <div class="o_Activity_sidebar mr-3">
```

Comment:
Ok if we handle it on another PR, but I feel it would look better if we increased the left spacing on the avatar for message, rather than reducing it for activities.

Reply:
ok. For reference:
> Add ml-1 and flex-shrink-0 to  o_Activity_sidebar.